CMakeLists.txt

```diff
@@ -78,11 +78,14 @@ set_target_properties(${LIBRARY_NAME}  PROPERTIES VERSION ${${VARS_PREFIX}_VERSI
                                        SOVERSION ${${VARS_PREFIX}_VERSION}
                                        PUBLIC_HEADER "${folder_header}")
 
-target_include_directories(${LIBRARY_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>"
-                                           "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/${${VARS_PREFIX}_INSTALL_INCLUDEDIR}>"
-                                           ${YARP_INCLUDE_DIRS}
-                                           ${iDynTree_INCLUDE_DIRS}
-                                           ${wholeBodyInterface_INCLUDE_DIRS})
+target_include_directories(${LIBRARY_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include/yarpWholeBodyInterface>"
```

Comment:
I expected errors, that's why I committed in a branch.

So... what I usually do (but I always get confused in the differences between install and build) is to use `"name.h"` for in-project include and `<libName/name.h>` for system (or external projects) include.

I think this (if it works of course) make it clearer the separation between project and external libraries.

I'll try to fix the compilation asap.